reactor-core/src/main/java/reactor/core/publisher/FluxRepeatWhen.java

```diff
@@ -211,7 +211,7 @@ public Object scanUnsafe(Attr key) {
 
 		RepeatWhenMainSubscriber<?> main;
 
-		final Sinks.Many<Long> completionSignal = Sinks.many().multicast().onBackpressureError();
+		final Sinks.Many<Long> completionSignal = Sinks.many().multicast().onBackpressureBuffer();
```

Comment:
`onBackpressureError()` was wrongly used and fails the same way in `master` if the queue is removed (e.g. by using `many().unsafe()`